styles/global.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css2?family=Besley:wght@400;700&display=swap');
```

Comment:
this font should be in another file called fonts. please use embedded fonts approach

Reply:
Good night engineer has already been modified to the requirements